src/bindings/Node.ml

```diff
@@ -47,25 +47,12 @@ module Buffer = struct
   let ofString = from
 end
 
-module type STREAM = sig
```

Comment:
This was functor was handy in implementing a lot of subtype relations in the Stream based APIs in Node.js like I/O, network etc